### DISCUSSION TITLE
More release builds for Windows would be great!

### DISCUSSION THREAD

**Discussion by Dr-InSide** [AlexandreRouma/SDRPlusPlus#759] [General]
I have tried to compile SDRpp on Windows itself, but always get an error somewhere. I'm not a C++ expert and it's getting annoying. SDR++ is a great program! So my wish: Please more frequent release builds for Windows or automatic nightly builds. Thanks a lot!

**Comment by the-gobleo-tinkerer** [comment]
> automatic nightly builds

Here: https://www.sdrpp.org/nightly/

**Reply by Dr-InSide** [reply]
Perfect, Thanks!!

**Comment by gbeyer3** [comment]
Sorry in advance for this noob question, but I don't have much experience with nightlies, or compiling my own software.  Having gone to the website, I went to nightly build, which redirected back to https://github.com/AlexandreRouma/SDRPlusPlus/actions/runs/2677721778

What do I download from there?   I don't see a .tar.    OR do I   $git clone . . . . what exactly?

Then I follow the build on Linux documentation.

Suppose I don't want to be bleeding edge, how would I find the nightly from, say, two weeks ago?

Thanks for your patience, and, as always, much credit and thanks to Alexandre!   This is an amazing project.

**Reply by the-gobleo-tinkerer** [reply]
On the page you linked, go down to Artifacts and click on the appropriate package for your distro & version. If none of them matches your system you'll probably need to build from source (which is usually not difficult).

You can find previous nightly builds in the [Actions](https://github.com/AlexandreRouma/SDRPlusPlus/actions) tab in this repo.

**Reply by gbeyer3** [reply]
Thank you very much, @shuyuan-liu !  Best regards, Greg
